mu4e/mu4e-main.el

```diff
@@ -228,9 +228,12 @@ clicked."
            when (not (and mu4e-main-hide-fully-read (eq unread 0)))
            concat (concat
                    ;; menu entry
-                   (mu4e--main-action-str
-                    (concat "\t* [j" key "] " name)
-                    (concat "j" key))
+		   (cond ((characterp key)
+			  (mu4e--main-action-str
+			   (concat "\t* [j" (string key) "] " name)
+			   (concat "j" (string key))))
+			 ((stringp key) (concat "\t* " key "  " name))
+			 (t (concat "\t*      " name)))
```

Comment:
Why so many spaces after `\t*`? I'm only asking as the space amount seems arbitrary, but perhaps for you it gives you the specific way you want it to look or something (for me it doesn't look as great if I were to take 2 of the spaces out).